mu4/scenes/inspector/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+#=============================================================================
+#  MuseScore
+#  Music Composition & Notation
+#
+#  Copyright (C) 2020 MuseScore BVBA and others
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License version 2.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+#=============================================================================
+get_filename_component(MODULE ${CMAKE_CURRENT_SOURCE_DIR} NAME)
+message("Configuring " ${MODULE})
+
+set(MODULE_QRC ${CMAKE_CURRENT_LIST_DIR}/view/inspector_resources.qrc )
+
+set(MODULE_QML_IMPORT ${CMAKE_CURRENT_LIST_DIR}/view/qml)
+
+file(GLOB_RECURSE inspector_SOURCES RELATIVE ${CMAKE_CURRENT_LIST_DIR} *.cpp)
+file(GLOB_RECURSE inspector_HEADERS RELATIVE ${CMAKE_CURRENT_LIST_DIR} *.h)
+file(GLOB_RECURSE inspector_ui_files RELATIVE ${CMAKE_CURRENT_LIST_DIR} *.ui)
```

Comment:
It is better to explicitly specify the files, as is done in other modules, otherwise the build system may not know that it needs to be updated.
It is easy to point files using the console, you need to make the window as narrow as possible, invoke the `ls` command, and copy the output.

Reply:
Done